platform_spec/chromium/manifest.json

```diff
@@ -19,7 +19,9 @@
 		"storage",
 		"webNavigation",
 		"tabs",
-		"declarativeNetRequestWithHostAccess"
+		"declarativeNetRequestWithHostAccess",
+		"webRequest",
+		"webRequestBlocking"
```

Comment:
> "webRequestBlocking"

This permission is no longer available
https://developer.chrome.com/docs/extensions/reference/webRequest/

Reply:
i know but I had to stop trying to make it work because I had some schoolwork to do so I'll continue on it Thursday the latest (unless someone else fixes it by then)
and ofc I'll remove that permission since it's expired